app-arch/zstd/zstd-1.5.7.recipe

```diff
@@ -109,39 +107,33 @@ TEST_REQUIRES="
 	cmd:python3
 	"
 
-PATCH()
-{
-	# avoid error on tests without static libs, we use LD_LIBRARY_PATH
-	sed '/build static library to build tests/d' -i build/cmake/CMakeLists.txt
-	sed 's/libzstd_static/libzstd_shared/g' -i build/cmake/tests/CMakeLists.txt
-}
-
 BUILD()
 {
-	cmake -B bld -S build/cmake -DCMAKE_BUILD_TYPE=Release \
-		$cmakeDirArgs \
-		-DCMAKE_INSTALL_BINDIR=$commandBinDir \
-		-DZSTD_BUILD_STATIC=OFF \
-		-DZSTD_BUILD_TESTS=ON \
-		-DZSTD_PROGRAMS_LINK_SHARED=ON
-	make -C bld $jobArgs
+	make DESTDIR="$installDestDir"
+		includedir="$includeDir" \
+		libdir="$libDir" \
+		bindir="$binDir" \
+		mandir="$manDir" \
+		$jobArgs
 }
 
 INSTALL()
 {
-	make -C bld install
+	make DESTDIR="$installDestDir" \
```

Comment:
DESTDIR is probably not needed for a non-cross, non-bootstrap package. Instead, I would add prefix here too for consistency.